plugins/kubernetes-common/package.json

```diff
@@ -40,11 +40,20 @@
   },
   "dependencies": {
     "@backstage/catalog-model": "workspace:^",
+    "@backstage/core-plugin-api": "workspace:^",
```

Comment:
Noting here again that this dependency can't exist

Reply:
fixed